mobify/documentation-theme
# WEB-1861: fixed issued with word wrap

Ticket: https://mobify.atlassian.net/browse/WEB-1861

Changed:
- Added word break style so that if there is one long text and it will automatic word break into lines so that will fix scroller issue.

How to test:
- `npm i`
- `npm link` 
- go to https://github.com/mobify/progressive-web-sdk
- `npm link @mobify/documentation-theme`
- `npm run docs:dev`
- Go to http://localhost:9000/latest/components/#!/Scroller 
- Verify that there is no scrolling issue on the page
- Verify other pages that it fixed scrolling issue
